### PR TITLE
[DOCS] Exception enhancements in 8.6

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2728 and makes various changes including:
- Chunked out a lot of the sections that were formerly part of the [Rule exceptions and value lists](https://www.elastic.co/guide/en/security/master/detections-ui-exceptions.html) topic in 8.5.
- Updated all general references to the Rule exceptions and value lists topic. 
- Changed all references to the Elastic Endpoint Security rule to "Endpoint Security" since that's what's shown in the UI now.

The following sections are the main items that need to be reviewed in this PR:

- [Elastic Security UI | Manage](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#_manage): Updated page name in second bullet and refreshed screenshot.
- [Rule exceptions](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html): Refreshed the whole topic. This topic mainly gives an overview of the exceptions feature.
  - [Create and manage value lists](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/value-lists-exceptions.html): The first paragraph in the topic is new. Everything else past that is unchanged.
  - [Add and manage exception items](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html): Lot's of minor updates to this topic:
    - Tweaked the topic's intro sentence to include that users can now create exceptions from the Rule Exceptions page.
    - Within the [Add exceptions to a rule](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#detection-rule-exceptions) section, I added a third bullet to the first step, added some additional info to step 2, and refreshed the step's screenshot. 
    - Within the [Add Elastic Endpoint exceptions](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#endpoint-rule-exceptions) section, I added a third bullet to the first step and refreshed the screenshot.
    - Refreshed the screenshot of the [nested conditions example](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#_nested_condition_example).
    - The [View and manage exceptions](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#manage-exception) section is new.
  - [Create and manage shared exception lists](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/shared-exception-lists.html): Expanded on what was formerly the [View and manage exception lists](https://github.com/elastic/security-docs/edit/main/docs/detections/detections-ui-exceptions.asciidoc) section:
    - The first three sections are new.
    - Refreshed content within the [View shared exception lists](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/shared-exception-lists.html#view-shared-exception-lists) and [Manage shared exception lists](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/shared-exception-lists.html#manage-exception-lists) sections.
- [Tune detection rules | Filter out uncommon application alerts](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/tuning-detection-signals.html#filter-rule-process): Refreshed steps 3-5, including the screenshot in step 4.
- [Tune detection rules | Tune rules detecting authorized processes](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/tuning-detection-signals.html#tune-authorized-processes): Refreshed screenshot in step 3 and the button name in step 4.
- [Manage detection rules | Manage rules](https://security-docs_2748.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html#manage-rules-ui): Added a short note about duplicating rules with exceptions.

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
Will need to update [this line](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#L384) in the get_doc_links.ts file to resolve the following error. Working on it with Lisa tomorrow. 


Screen Shot 2022-11-28 at 2 45 22 PM

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc:48]:
```suggestion
. In the *Add rule exception* flyout, name the exception item and add conditions that define when the exception prevents alerts. When exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria is met.
```

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc:95]:
```suggestion
. Click *Add rule exception*. 
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [inline docs/detections/shared-exception-lists.asciidoc:24]:
```suggestion
. In the *Add rule exception* flyout, name the exception item and add conditions that define when the exception prevents alerts. When exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria is met.
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [inline docs/detections/shared-exception-lists.asciidoc:59]:
```suggestion
. Click *Add rule exception*.
```

**Review by nastasha-solomon — COMMENTED** [review]
casing edits

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [inline docs/detections/shared-exception-lists.asciidoc:4]:
```suggestion
Shared exception lists allow you to group exceptions together and then apply them to a set of rules. Use the Rule Exceptions page to set up shared exception lists.  
```

File: docs/detections/rules-ui-manage.asciidoc
Comment (peluja1012) [inline docs/detections/rules-ui-manage.asciidoc:139]: I think it's worth explaining what happens when they choose to "duplicate the rule with its exceptions" and the rule contains a shared exception list. In this case, the shared exception list reference will be kept. In other words, the duplicated rule will also have a reference to the same shared list.

**Review by peluja1012 — APPROVED** [review]
Great job making this complex feature easily digestible, Nastasha! The content looks good to me aside from a minor comment.

**Review by joepeeples — COMMENTED** [review]
Outstanding, massive work! 🔥 Excellent job organizing and expanding these docs, and making everything clear and straightforward.

One overall question I've been pondering is when "endpoint exception" should be capitalized as "Endpoint exception." I think capitalized "Endpoint exception" is usually more appropriate because those exceptions are specifically related to Elastic Endpoint's activity, vs. events generally happening on endpoints/hosts. 

I didn't add any edits for this, because the docs have historically been a bit inconsistent on this term already, so mostly it's nothing that this PR has introduced or needs to fix. I can open a backlog issue to revisit terms like this in general, as part of our gradual tightening-up of terminology. But for this PR, the distinction between regular rule exceptions and Endpoint exceptions is clear, so I think scrutinizing big E's or little e's can wait until after merging this for release. 😄

File: docs/detections/shared-exception-lists.asciidoc
Comment (benironside) [inline docs/detections/shared-exception-lists.asciidoc:25]: Maybe consider cutting this line? It feels a but redundant with line 23, but also a little confusing because it's a bit different. I feel like line 23 could go right into the instructional bullet points starting on line 27.

File: docs/detections/shared-exception-lists.asciidoc
Comment (benironside) [inline docs/detections/shared-exception-lists.asciidoc:30]:
```suggestion
TIP: You can add exceptions to an empty shared exception list by expanding the list or viewing its details page and clicking *Create rule exception*. After creating an exception, you can associate the shared exception list with rules. Refer to <<link-shared-exception-lists>> to learn more.
```

File: docs/detections/value-list-exceptions.asciidoc
Comment (benironside) [inline docs/detections/value-list-exceptions.asciidoc:30]:
```suggestion
act as delimiters.
```

File: docs/detections/value-list-exceptions.asciidoc
Comment (benironside) [inline docs/detections/value-list-exceptions.asciidoc:15]:
```suggestion
TIP: You can also use a value list as the <<indicator-value-lists,indicator match index>> when creating an indicator match rule.
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (benironside) [inline docs/detections/shared-exception-lists.asciidoc:128]:
```suggestion
* If an exception list is linked to any rules, you'll get a warning asking you to confirm the deletion. 
```

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [inline docs/detections/add-exceptions.asciidoc:80]:
```suggestion
. Choose to add the exception to a rule or a shared exception list. 
```
maybe?

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [inline docs/detections/add-exceptions.asciidoc:101]:
```suggestion
Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Endpoint Security rule or by adding them as actions on alerts generated by the Endpoint Security rule. {elastic-endpoint} alerts have the following fields:
```

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [inline docs/detections/add-exceptions.asciidoc:114]:
```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful
```

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [inline docs/detections/add-exceptions.asciidoc:130]:
```suggestion
.. Go to the rule details page (*Manage* -> *Rules*), and then search for and select the Elastic *Endpoint Security* rule.
```

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [inline docs/detections/add-exceptions.asciidoc:195]: I know this tip was here before, but I'm trying to figure out how it connects to anything else on the page? No other mentions of `code_signature.subject_name`. Maybe cut it?

**Review by benironside — COMMENTED** [review]
GREAT WORK! Left some suggestions, mostly very minor, let me know if you have any questions!

File: docs/detections/detections-ui-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/detections-ui-exceptions.asciidoc:17]: @joepeeples good idea! Your proposed addition sounds and looks accurate. @peluja1012 @dhurley14 would you mind reviewing the new addition Joe suggested?

File: docs/detections/detections-ui-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/detections-ui-exceptions.asciidoc:32]: Gotcha, that makes sense - thanks for explaining! I also agree with your recommended change. It removes any suggestion that users can apply an exception list to a logically/manually grouped batch of rules.

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:55]:
```suggestion
+
Add conditions that define when the exception prevents alerts:

  .. *Field*: Select a field to identify the event being filtered.
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc:67]:
```suggestion
. (Optional) Enter a comment describing the exception.

. (Optional) *Close all alerts that match this exception and were generated by this rule*:
Closes all alerts that match the exception's conditions and were generated only by the current rule.
+
. Click *Add rule exception*.
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc:106]: Yeah, this was pre-existing content so I figured it was still relevant/accurate. I'm onboard with removing the link and suggestion to enter a query though.

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc:25]: I also went back and forth on this for the reasons you listed. I ultimately decided to keep it so that it matched the structure the previous and following sections used (e.g., title, lead-in sentence, then steps). I'm still leaning towards keeping it to maintain that consistency, but agree with you that the lead-in sentence is unclear. How about something simpler like:

```suggestion
Add exception items:
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc]: Agree that we can remove this - it's pretty obvious 😅

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:114]: Hm, I'm split on this. I kind of want to keep "Endpoint" in the second sentence because I think the repetition helps emphasize the practice we're recommending. How about:

```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful Endpoint
```

Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:114]: Hm, I'm split on this. I kind of want to keep "Endpoint" in the second sentence because I think the repetition helps emphasize the practice we're recommending. How about:

```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful Endpoint
```

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:195]: I think this is mainly a tip about the example in lines 172-192. I'm ok with removing it though if it's not all that relevant to exceptions in general. :)

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc]:
```suggestion
. Choose to add the exception to a rule or a shared exception list. 
```

Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc]: Re-adding @benironside comment from [here](https://github.com/elastic/security-docs/pull/2748/files/add58aa1c620651778ee559e27fc12c4ed7f385f#r1038620714).

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc]: Re-adding Ben's comment from [here](https://github.com/elastic/security-docs/pull/2748/files/add58aa1c620651778ee559e27fc12c4ed7f385f#r1038622933).

```suggestion
.. Go to the rule details page (*Manage* -> *Rules*), and then search for and select the Elastic *Endpoint Security* rule.
```

**Review by nastasha-solomon — COMMENTED** [review]
Re-adding comments that didn't get applied.

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc]: Suggestion applied

File: docs/detections/shared-exception-lists.asciidoc
Comment (joepeeples) [reply docs/detections/shared-exception-lists.asciidoc]: The "Close all alerts..." step might still need to be revised to match other steps, with an action verb such as "(Optional) Select *Close all alerts...* to close all alerts blah blah"

Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc]: oh yeah, good catch. I'll make that correction in my next round of updates.

File: docs/detections/add-exceptions.asciidoc
Comment (jmikell821) [inline docs/detections/add-exceptions.asciidoc:142]:
```suggestion
NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated with the Endpoint Security rule any rules with the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.  
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (jmikell821) [inline docs/detections/shared-exception-lists.asciidoc:30]:
```suggestion
TIP: You can add exceptions to an empty shared exception list by expanding the list, or viewing its details page and clicking *Create rule exception*. After creating an exception, you can associate the shared exception list with rules. Refer to <<link-shared-exception-lists>> to learn more.
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (jmikell821) [inline docs/detections/shared-exception-lists.asciidoc:117]:
```suggestion
You can edit, export, import, and delete shared exception lists from the Rule Exceptions page.  
```

File: docs/detections/shared-exception-lists.asciidoc
Comment (jmikell821) [reply docs/detections/shared-exception-lists.asciidoc:25]: > I also went back and forth on this for the reasons you listed. I ultimately decided to keep it so that it matched the structure the previous and following sections used (e.g., title, lead-in sentence, then steps). I'm still leaning towards keeping it to maintain that consistency, but agree with you that the lead-in sentence is unclear. How about something simpler like:

@nastasha-solomon I like this option. 👍

**Review by jmikell821 — COMMENTED** [review]
Slight comments/suggestions, but overall this looks great. Thanks for all your hard work on this PR! 🌟

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc]: Thanks for catching this! I didn't notice it before. I'm pretty sure "agent" isn't needed here, right @joepeeples ?

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:141]:
```suggestion
NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated with the Endpoint Security rule and any rules with the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.
```

Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:141]: Re-added "and" because I think it was accidentally removed.

File: docs/detections/shared-exception-lists.asciidoc
Comment (nastasha-solomon) [reply docs/detections/shared-exception-lists.asciidoc:117]: Oops, forget the oxford comma here 😅

File: docs/detections/add-exceptions.asciidoc
Comment (benironside) [reply docs/detections/add-exceptions.asciidoc:116]: Love it

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc:197]:
```suggestion

```

Comment (nastasha-solomon) [reply docs/detections/add-exceptions.asciidoc:197]: Removing as per suggestion [here](https://github.com/elastic/security-docs/pull/2748#discussion_r1039121452).

File: docs/detections/add-exceptions.asciidoc
Comment (nastasha-solomon) [inline docs/detections/add-exceptions.asciidoc:197]:
```suggestion

```

**Pull request merged**
